### PR TITLE
Updated drmingw to 0.9.5

### PR DESCRIPTION
**Pull request by hmartinez82**
Notes:
- Removed unused patch
- Renamed the first patch to have the 0001 prefix
- The upstream repo now has two git submodules and they are not included in the .tar.gz file for the release so switching to git cloning the specific tag/release plus initializing the submodules was required.

### CODE REVIEW

**Comment by hmartinez82** [comment]
Investigating...

**Comment by kmilos** [comment]
Would it be possible to debundle at least zlib? It's such a common system lib, and we should probably request the something like the "USE_SYSTEM_ZLIB" CMake option upstream anyway...

I guess libdwarf must stay bundled until we package it for MSYS2 at some point.

File: mingw-w64-drmingw/PKGBUILD [hunk 1]
```diff
@@ -4,8 +4,8 @@
 _realname=drmingw
 pkgbase="mingw-w64-${_realname}"
 pkgname=(${MINGW_PACKAGE_PREFIX}-${_realname})
-pkgver=0.9.3
-pkgrel=3
+pkgver=0.9.5
+pkgrel=4
```
Comment (MehdiChinoune) [inline mingw-w64-drmingw/PKGBUILD:8]:
```suggestion
pkgrel=1
```

**Comment by asmwarrior** [comment]
Hi, drmingw 0.9.7 is released, and this release fixes a bug that a RPT can't be saved, see here: [ExcHndl.dll can't print the crash call stack under Windows 7](https://github.com/jrfonseca/drmingw/issues/71), can you update in msys2 packages? Thanks.

**Comment by MehdiChinoune** [comment]
> Hi, drmingw 0.9.7 is released, and this release fixes a bug that a RPT can't be saved, see here: [ExcHndl.dll can't print the crash call stack under Windows 7](https://github.com/jrfonseca/drmingw/issues/71), can you update in msys2 packages? Thanks.

see: #13106
Although Windows 7 will be dropped soon #10088

**Comment by asmwarrior** [comment]
Hi, that issue title(mention Windows 7) is misleading, because I use Windows 7. But the actual fix is in a file path handling code, so it is not specific to Windows 7.

See the actual code here:

[exchndl: Fix path truncation. · jrfonseca/drmingw@d64a9c7](https://github.com/jrfonseca/drmingw/commit/d64a9c7e693a2ac9f6efb3c24d9a9572bb132aac)

**Comment by asmwarrior** [comment]
> > Hi, drmingw 0.9.7 is released, and this release fixes a bug that a RPT can't be saved, see here: [ExcHndl.dll can't print the crash call stack under Windows 7](https://github.com/jrfonseca/drmingw/issues/71), can you update in msys2 packages? Thanks.
> 
> see: #13106 Although Windows 7 will be dropped soon #10088

Thanks for your pull request!

**Pull request merged**